src/game/behaviors/bowling_ball.inc.c

```diff
@@ -1,5 +1,9 @@
 // bowling_ball.inc.c
 
+// There is no good way to handle this
+#include "levels/bob/header.h"
+#include "levels/ttm/header.h"
+
```

Comment:
Why not just add this to `obj_behaviors.c` instead? If the argument is in favor of syntax highlighting, that's out of scope of this PR (tbh this might just get in the way of addressing that cleanly if anything).

Reply:
Same reason, I would like to have no "levels/..." includes at all. I left it here because thats the only place this is referenced from.